src/inventory_ui.h

```diff
@@ -124,34 +178,45 @@ class inventory_column
         bool is_selected_by_category( const inventory_entry &entry ) const;
 
         const inventory_entry &get_selected() const;
-        std::vector<inventory_entry *> get_all_selected() const;
-        const std::vector<inventory_entry> &get_entries() const {
-            return entries;
-        }
+        std::vector<inventory_entry *> get_all_selected();
 
-        inventory_entry *find_by_invlet( long invlet ) const;
+        inventory_entry *find_by_invlet( long invlet );
 
         void draw( WINDOW *win, size_t x, size_t y ) const;
 
         void add_entry( const inventory_entry &entry );
-        void add_entries( const inventory_column &source );
+        void remove_entry( const inventory_entry &entry );
+        void move_entries_to( inventory_column &dest );
+        void clear();
 
         void set_multiselect( bool multiselect ) {
             this->multiselect = multiselect;
         }
 
-        void set_custom_colors( bool custom_colors ) {
-            this->custom_colors = custom_colors;
-        }
-
         void set_mode( navigation_mode mode ) {
             this->mode = mode;
         }
 
-        virtual size_t get_width() const;
-        virtual void prepare_paging( size_t new_entries_per_page = 0 ); // Zero means unchanged
+        void set_visibility( bool visibility ) {
+            this->visibility = visibility;
+        }
 
-        virtual void on_action( const std::string &action );
+        void set_width( size_t width );
+        void set_height( size_t height );
+        size_t get_width() const;
+        size_t get_height() const {
+            return entries_per_page;
+        }
+        /// Expands the column to fit the new entry
```

Comment:
`/** */` seems to be the most common variant but I presume this works also with doxygen?